src/components/Dashboard/Student/Profile/StudentProfile.css

```diff
@@ -0,0 +1,23 @@
+.student-profile{
+    width: 83.5;
+    position: absolute;
+    right: 0;
+    box-sizing: border-box;
+
+}
+
+.right-profile>.about{
+    height: auto !important;
```

Comment:
i see important in many places in many css files. please recheck once (low priority)